src/reducers/root.js

```diff
@@ -1,6 +1,6 @@
 import { routerReducer } from "react-router-redux";
 import { reducer as formReducer } from "redux-form";
-import { reducer as LoginReducer } from "./login";
+import * as LoginReducer from "./login";
```

Comment:
Do you sure this work? If LoginReducer return not default, you will need `{... as ...}`.